depthai_ros_driver/include/depthai_ros_driver/depthai_common.hpp

```diff
@@ -42,27 +38,16 @@
   #include <depthai_ros_msgs/msg/objects.hpp>
   #include <depthai_ros_msgs/msg/auto_focus_ctrl.hpp>
   #include <camera_info_manager/camera_info_manager.hpp>
-  #include <rclcpp/node.hpp>
-
-  #define ROS_LOGGER(...) RCUTILS_LOG_WARN_NAMED("depthai_common", __VA_ARGS__)
 
 using ImageMsg = sensor_msgs::msg::Image;
 using CompressedImageMsg = sensor_msgs::msg::CompressedImage;
 using ObjectsMsg = depthai_ros_msgs::msg::Objects;
 using ObjectMsg = depthai_ros_msgs::msg::Object;
 using CameraInfoMsg = sensor_msgs::msg::CameraInfo;
 using HeaderMsg = std_msgs::msg::Header;
-using ROSNodeHandle = std::shared_ptr<rclcpp::Node>;
-
-using ObjectPubPtr = rclcpp::Publisher<ObjectMsg>::SharedPtr;
-using ObjectsPubPtr = rclcpp::Publisher<ObjectsMsg>::SharedPtr;
-using ImagePubPtr = rclcpp::Publisher<ImageMsg>::SharedPtr;
-using ComImagePubPtr = rclcpp::Publisher<CompressedImageMsg>::SharedPtr;
-using StreamPub = std::variant<ObjectsPubPtr, ObjectPubPtr,
-    ImagePubPtr, ComImagePubPtr>;
-using CameraInfoPub = rclcpp::Publisher<CameraInfoMsg>::SharedPtr;
-using RosTime = rclcpp::Time;
-using RosDuration = rclcpp::Duration;
+using AutoFocusCtrlMsg = depthai_ros_msgs::msg::AutoFocusCtrl;
```

Comment:
Thoughts on adding a macro:
```cpp
#if defined(ROS1)
#define MSG_TYPE(PACKAGE, NAME) PACKAGE::NAME
#elif defined(ROS2)
#define MSG_TYPE(ROS1_TYPE, ROS2_TYPE) PACKAGE::msg::NAME
#endif
```

This will make it:
```cpp
using AutoFocusCtrlMsg = MSG_TYPE(depthai_ros_msgs, AutoFocusCtrl);
```

or alternatively, something like:
```cpp
using AutoFocusCtrlMsg = ROS_AGNOSTIC_TYPE(depthai_ros_msgs::AutoFocusCtrl, depthai_ros_msgs::msg::AutoFocusCtrl);
```